### DISCUSSION TITLE
Vercel hosting - What defines hobby?

### DISCUSSION THREAD

**Discussion by Gusterson** [vercel/vercel#4571] [Help] [answered]
Looking for clarity on what defines the free "hobby" tier of pricing.

Let's say I have a blog that has ads that generate enough money to pay for hosting (barely). Is this commercial or hobby?

Let's say that blog, being super-informative and engaging racks up 1,000 hits a day, and starts bringing in some ok money from ads and patreon sponsorship. Is that hobby or commercial?

Let's say that blog gets enough interest to sell T-shirts on it. I'm still the only developer / content creator. Is that commercial or hobby?

**Comment by mcsdevv** [comment, marked as answer]
Hey @Gusterson, I think a fair way to put this would be any site that displays ads or charges money falls under the commercial bracket and thus requires a Pro plan.

It's extremely likely that a very popular blog, far in excess of 1000 hits a day (we host a few), could be run under the Hobby plan.

**Reply by Gusterson** [reply]
Thanks for this. So the distinction is not whether I'm a commercial entity (company) but whether I have some form of revenue generating mechanism on the site -- no matter how small.

So if I solicit donations (patreon, etc) but don't run ads, then we're still looking at the "hobby" plan?

**Reply by mcsdevv** [reply]
That would be correct!

**Reply by davedavedave123** [reply]
I have a follow up question sort of along these lines.

I build websites for local businesses. Is the $20 a month per team member charged for each website? Or is that charged to my team and I can build x amount of websites for that price?

**Reply by mcsdevv** [reply]
Totally your call @davedavedave123 - you can build everything from within a single Vercel team, or you could have a team for each. Nothing on the platform will stop you from doing either, it's very much personal preference although the latter option would almost certainly cost you more.

**Reply by davedavedave123** [reply]
Ok great, thanks!
